content/en/docs/tasks/administer-cluster/certificates.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Certificates
+content_type: concept
```

Comment:
Would you change the `content_type` to: task
```
content_type: task
```

Reply:
oops my bad! done